docs/content/preview/explore/observability/_index.md

```diff
@@ -41,19 +41,19 @@ YugabyteDB exports various [metrics](../../launch-and-manage/monitor-and-alert/m
 
 Monitoring involves continuously checking the system's health and performance and notifying stakeholders if any issues arise. For this, you can set up automated alerts based on predefined thresholds or conditions. All metrics exposed by YugabyteDB are exportable to third-party monitoring tools like [Prometheus](./prometheus-integration/macos/) and [Grafana](./grafana-dashboard/grafana/) which provide industry-standard alerting functionalities.
 
-{{<tip>}}
+{{<lead link="../../yugabyte-cloud/cloud-monitor/cloud-alerts/">}}
 Both [YugabyteDB Anywhere](../../yugabyte-platform/alerts-monitoring/) and [YugabyteDB Managed](../../yugabyte-cloud/cloud-monitor/cloud-alerts/) provide a full suite of alerting capabilities for monitoring.
-{{</tip>}}
+{{</lead>}}
 
 ## Visualization and analysis
 
 YugabyteDB provides dashboards that include charts, graphs, and other visual representations of the system's state and performance. [yugabyted](../../reference/configuration/yugabyted/) starts a web-UI on port 15433 that displays different charts for various metrics.
 
 You can also export the metrics provided by YugabyteDB onto third-party visualization tools like [Prometheus](./prometheus-integration/macos/) and [Grafana](./grafana-dashboard/grafana/) as per the needs of your organization.
 
-{{<tip>}}
+{{<lead link="../../yugabyte-cloud/cloud-monitor/overview/">}}
```

Comment:
same thing here.. not sure if we want to have leads or tip in cases where there multiple links.